examples/integrations/outlook/connector.js

```
@@ -711,6 +773,117 @@ function getContact(http, headers, params) {
+        fileId: file.fileId,
+        url: file.url,
+      });
+    } else {
+      var downloadUrl =
+        GRAPH_BASE_URL +
+        '/me/messages/' +
+        params.messageId +
+        '/attachments/' +
+        att.id +
+        '/$value';
+      var file = files.download(downloadUrl, {
```

Biome (2.3.14)

[error] Shouldn't redeclare 'file'. Consider to delete it or rename it.

'file' is defined here:


(lint/suspicious/noRedeclare)